src/shared/components/iframe-runtime.tsx

```diff
@@ -14,31 +12,36 @@ interface ILogRequest {
 }
 
 interface IProps {
-  url: string;
-  id: string;
-  authoredState: any;
+  authoredState?: any;
+  id?: string;
+  iframeStyling?: any;
   interactiveState: any;
-  setInteractiveState: (state: any) => void;
-  report?: boolean;
+  logRequestData?: Record<string, unknown>;
   navImageUrl?: string;
   navImageAltText?: string;
+  report?: boolean;
+  scaffoldedQuestionLevel?: number;
+  url?: string;
```

Comment:
Is there a case where the url is not set? 
It seems like the iframe would always need an url.

Reply:
looks good